src/genie/libs/parser/iosxe/tests/ShowVrrp/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,41 @@
+expected_output = { 
+     {
+    "interface": {
+        "GigabitEthernet1": {
+            "group_number": {
+                "Group 100": {
+                    "auth_text": '"hash"',
+                    "device_advertise_interval": "1.000",
+                    "master_adv_int": "1.000",
+                    "master_down_int": "3.531",
+                    "master_ip_address": "192.168.1.233",
+                    "master_router": "local",
+                    "master_router_priority": "120",
+                    "preemption_state": "enabled",
+                    "priority": "120",
+                    "track_object_decrement_value": 30,
+                    "track_object_number": "1",
+                    "track_object_state": "Up",
```

Comment:
these lines are under `VRRS GRoup name DC_LAN`. we respect indentation so would suggest changing to:
```python
"vrrf_group_name": {
    "DC_LAN": {
        "track_object": {
            1: {
                "state": "Up",
                "decrement": 30,
```
corresponding output:
```
  VRRS Group name DC_LAN
    Track object 1 state Up decrement 30
```

Reply:
still seeing like below.
```
                    "vrrs_group_name": {
                        "DC_LAN": {
                            "track_object": {
                                1: {
                                    "track_object_decrement_value": 30,
                                    "track_object_state": "Up",
                                }
                            }
                        }
                    },
```
but `state` and `decrement` is good since we know it's `track_object` from parent level.
```
                    "vrrs_group_name": {
                        "DC_LAN": {
                            "track_object": {
                                1: {
                                    "decrement": 30,
                                    "state": "Up",
                                }
                            }
                        }
                    },
```

Reply:
Hi Takashi,

Thank you for reviewing my changes. I'm sorry I missed it. I've addressed the values and the output looks like below:
 "vrrs_group_name": {
                        "DC_LAN": {
                            "track_object": {
                                1: {
                                    "decrement": 30,
                                    "state": "Up",
                                }
                            }
                        }
                    },
Thanks,

Juhi